fem/coefficient.hpp

```diff
@@ -419,18 +419,20 @@ class GridFunctionCoefficient : public Coefficient
     where T is the transformation rule, and Q1/Q2 are the parent coefficients.*/
 class TransformedCoefficient : public Coefficient
 {
+   using t1_f = std::function<double(double)>;
```

Comment:
In order to be consistent with `FunctionCoefficient`, `VectorFunctionCoefficient`, and `MatrixFunctionCoefficient`, I would either remove the `using` and use the full types everywhere or make this `using` change for all the cases.